flink-end-to-end-tests/flink-tpcds-test/src/main/java/org/apache/flink/table/tpcds/utils/TpcdsResultComparator.java

```diff
@@ -52,26 +54,30 @@ public class TpcdsResultComparator {
     private static final String RESULT_SUFFIX = ".ans";
     private static final double TOLERATED_DOUBLE_DEVIATION = 0.01d;
 
-    public static void main(String[] args) throws Exception {
+    public static void main2(String[] args) throws Exception {
```

Comment:
I'm curious that why you rename it main2?

Reply:
BTW, I see no one call `TpcdsResultComparator`  this class, why need we to modify it?

Reply:
> I'm curious that why you rename it main2?
That's a typo... reverted it.
> BTW, I see no one call TpcdsResultComparator this class, why need we to modify it?
You can see it in test_tpcds.sh.